internal/flags/options.go

```diff
@@ -64,4 +64,9 @@ type Options struct {
 
 	// TestingT runs scenarios as subtests.
 	TestingT *testing.T
+
+	// FeatureContents allows passing in each feature manually
+	// where the contents of each feature is stored as a byte slice
+	// in a map entry
+	FeatureContents map[string][]byte
```

Comment:
Let's make this a slice of structs to improve clarity of data items.
Something like.
```
	// Features allow passing in each feature manually
	// where the contents of each feature is stored as a byte slice
	// in a map entry
	Features []Feature
}

// Feature contains path and contents of a gherkin file.
type Feature struct {
	Path    string
	Content []byte
}
```